internal/controller/http/common.go

```diff
@@ -24,21 +24,21 @@ import (
 // Ping handles the request to /ping endpoint. Is used to test if the service is working
 // It returns a response as specified by the V2 API swagger in openapi/common
 func (c *RestController) Ping(writer http.ResponseWriter, request *http.Request) {
-	response := commonDTO.NewPingResponse()
+	response := commonDTO.NewPingResponse(c.serviceName)
```

Comment:
this kind of signature change will break all the original code after upgrade go-mod-core-contracts

Reply:
@cloudxxx8 , this is using the upgraded go-mod-core-contracts and adjusting for the signature change we agreed was ok since it is only use by our internal code.

Reply:
@lenny-intel go-mod-core-contracts is a public module, but if you have discussed and agreed, I am fine with this change.

Reply:
@cloudxxx8 , you agreed here: https://github.com/edgexfoundry/go-mod-core-contracts/pull/695#discussion_r791833007 
:-)